lib/bots/versionbot.ts

```diff
@@ -833,7 +866,8 @@ export class VersionBot extends ProcBot {
             moddedFiles.push(`CHANGELOG.md`);
 
             // Now we get the new version from the CHANGELOG (*not* the package.json, it may not exist).
-            return exec(`cat ${versionData.fullPath}${_.last(moddedFiles)}`).then((contents: string) => {
+            return fsReadFile(`${versionData.fullPath}${_.last(moddedFiles)}`).call('toString')
```

Comment:
If you pass an encoding to `readFile` I think it'll come back with a string for you, so you can drop the `toString`